bin/git-sync-mirror.sh

```diff
@@ -180,15 +180,22 @@ if ! whoami &> /dev/null; then
 fi
 
 # Create local repositories
-LOCAL_REPO_SRC="$(mktemp -d)"
-LOCAL_REPO_DST="$(mktemp -d)"
+LOCAL_REPO_SRC="${LOCAL_REPO_SRC:-"$(mktemp -d)"}"
+LOCAL_REPO_DST="${LOCAL_REPO_DST:-"$(mktemp -d)"}"
+
+if [ -z "$(ls -A ${LOCAL_REPO_SRC})" ]; then
```

Comment:
using `ls -A` to detect if something is a git repository, is not correct.

```suggestion
if [ ! -d "${LOCAL_REPO_SRC}/.git" ]; then
```

Reply:
I see. My only logic was that only clone if its emtpy. But checking for .git makes more sense.